src/test/java/org/candlepin/subscriptions/subscription/export/PerformanceSubscriptionDataExporterServiceIT.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.subscription.export;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+
+import com.redhat.cloud.event.apps.exportservice.v1.Format;
+import com.redhat.swatch.configuration.util.MetricIdUtils;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import org.candlepin.clock.ApplicationClock;
+import org.candlepin.subscriptions.db.model.BillingProvider;
+import org.candlepin.subscriptions.db.model.Subscription;
+import org.candlepin.subscriptions.db.model.SubscriptionMeasurementKey;
+import org.candlepin.subscriptions.export.BaseDataExporterServiceTest;
+import org.candlepin.subscriptions.test.ExtendWithSwatchDatabase;
+import org.hibernate.SessionFactory;
+import org.hibernate.StatelessSession;
+import org.hibernate.Transaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
+
+@Disabled
```

Comment:
The test is disabled on purpose because it might be problematic in environments like Jenkins CI and in the end, this will be coped by the performance team as part of another ticket.
I added it to confirm that the parameter "dev.resteasy.entity.file.threshold" worked fine (if you remove the `@Disabled` annotation, and comment the setting of the parameter in the BootApplication.java class, the test will fail).

Reply:
How do you feel about verifying the actual body size?

Reply:
@awood PR updated with this.